examples/app/simple_hdf5/create_hdf5.py

```diff
@@ -0,0 +1,25 @@
+import h5py
+import numpy as np
+
+def getData(f, name):
+    shape = f[name].shape
+    # Empty array
+    data = np.empty(shape, dtype=np.float64)
+    # read_dicect to empty arrays
```

Comment:
1) typo
2) I don't quite understand this comment and am not sure it's necessary, the following line is fairly self-explanatory

This would be very self-explanatory if `data` was renamed `empty_data`